db/migrate/20151116114008_retire_links_column_from_links_sets.rb

```diff
@@ -0,0 +1,5 @@
+class RetireLinksColumnFromLinksSets < ActiveRecord::Migration
+  def change
+    rename_column :link_sets, :links, :legacy_links
+  end
+end
```

Comment:
I see – I guess that's reasonable. We just need to be extra careful that the migration works correctly.